Shared/Tests.Shared/TestObjects.cs

```diff
@@ -209,6 +210,25 @@ public class Owner : RealmObject
     }
 
     [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass")]
+    public class Employee : RealmObject
+    {
+        public string Name { get; set; }
+
+        public IList<Employee> Reports { get; }
+
+        [Backlink(nameof(Employee.Reports))]
+        public IQueryable<Employee> Bosses { get; }
+
+        public Employee Boss
+        {
+            get
+            {
+                return Bosses.ToList().SingleOrDefault();
```

Comment:
I wrote it that way to show a workaround for the period up until it is fixed. Would you like a comment on there to make that clear (sorry I should have had one)?